lib/commands/fastboot.js

```diff
@@ -15,27 +16,29 @@ module.exports = {
     { name: 'assets-path', type: String, default: 'dist' }
   ],
 
-  runCommand: function(appName, options) {
-    var commandOptions = this.commandOptions;
-    var outputPath = commandOptions.outputPath;
-    var assetsPath = commandOptions.assetsPath;
+  startServer: function(options) {
     var ui = this.ui;
+    var self = this;
 
-    ui.writeLine("Installing FastBoot npm dependencies");
+    if (this._fastBootListener) {
```

Comment:
I recall some issues in the past where programmatically restarting a server required I also flush require.cache otherwise stale modules were loaded.  I could be wrong, just jotting down something that came to mind